Assets/Scripts/StandardSamples/UI/Storage/Title/UITitleStorageMenu.cs

```diff
@@ -64,13 +64,12 @@ protected override void Awake()
 
         private void Start()
         {
-            TitleStorageManager = EOSManager.Instance.GetOrCreateManager<EOSTitleStorageManager>();
         }
 
         protected override void OnDestroy()
         {
             base.OnDestroy();
-            EOSManager.Instance.RemoveManager<EOSTitleStorageManager>();
+            EOSManager.Instance.RemoveManager<TitleStorageService>();
```

Comment:
Should this be removed through this way, now that we use Singletons?

Reply:
Yep, done.